examples/grpc.core/Examples.GrpcCore.AspNetCore/Examples.GrpcCore.AspNetCore.csproj

```diff
@@ -1,23 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>$(DefaultTargetFrameworkForExampleApps)</TargetFramework>
+    <NoWarn>$(NoWarn);CS8981</NoWarn>
```

Comment:
Can we remove this warning by updating packages? As it is example project, we can update to the latest packages.
Grpc.Tools generates classes with better names for .NET8. Tested shortly on my env, it will be great if you can double check this.

```patch
diff --git a/examples/grpc.core/Examples.GrpcCore.AspNetCore/Examples.GrpcCore.AspNetCore.csproj b/examples/grpc.core/Examples.GrpcCore.AspNetCore/Examples.GrpcCore.AspNetCore.csproj
index dafc9f79..0ca7c555 100644
--- a/examples/grpc.core/Examples.GrpcCore.AspNetCore/Examples.GrpcCore.AspNetCore.csproj
+++ b/examples/grpc.core/Examples.GrpcCore.AspNetCore/Examples.GrpcCore.AspNetCore.csproj
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>$(DefaultTargetFrameworkForExampleApps)</TargetFramework>
-    <NoWarn>$(NoWarn);CS8981</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="OpenTelemetry.Exporter.Console" Version="$(OpenTelemetryCoreLatestVersion)" />
     <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="$(OpenTelemetryCoreLatestVersion)" />
-    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.8.1" />
-    <PackageReference Include="Grpc.Tools" Version="1.17.0" />
-    <PackageReference Include="Grpc" Version="[2.32.0,3.0)" />
+    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.9.0" />
+    <PackageReference Include="Grpc.Tools" Version="2.66.0" />
+    <PackageReference Include="Grpc" Version="[2.46.6,3.0)" />
   </ItemGroup>
 
   <ItemGroup>

```

Reply:
Thank you!